public/6.html

```diff
@@ -53,14 +53,14 @@
 function removeColorFromList(color) {
   $('.colorList').find('[data-id="'+color.id+'"]').remove();
   if ($('.colorList li').length === 0)  {
-    $('html').css('background', 'transparent');
+    $(document.body).css('background', 'transparent');
   }
 }
 function updateColorOnList(color) {
   $('.colorList').find('[data-id="'+color.id+'"]')
     .text(color.name+' (id: '+color.id+')');
   if (color.id === lastColorId) {
-    $('html').css('background', color.name);
+    $(document.body).css('background', color.name);
   }
 }
```

Comment:
@gr2m I realize I was working off a branch that had these changes in place and previously made a PR for. https://github.com/hoodiehq/hoodie-camp-tutorial/pull/29